airbyte_cdk/test/mock_http/request.py

```diff
@@ -72,7 +72,11 @@ def _to_mapping(
         elif isinstance(body, bytes):
             return json.loads(body.decode())  # type: ignore  # assumes return type of Mapping[str, Any]
         elif isinstance(body, str):
-            return json.loads(body)  # type: ignore  # assumes return type of Mapping[str, Any]
+            try:
```

Comment:
Without this addition, depending on the order of evaluation, test `test_given_on_match_is_mapping_but_not_input_when_matches_then_return_false` would fail